src/labels.js

```diff
@@ -69,10 +49,18 @@ class Labels {
     }
   }
 
-  static initMetadata (wallet) {
-    return Metadata.fromMasterHDNode(wallet._metadataHDNode, METADATA_TYPE_LABELS);
+  get readOnly () {
+    return false;
+  }
+
+  syncWallet () {
+    const syncWallet = () => {
+      this._syncWallet(() => Promise.resolve(), (e) => Promise.reject(e));
```

Comment:
These callbacks aren't doing anything. To have an effect `syncWallet`  (defined on 57) would have to return a promise.

Aside: should Labels be implementing an public `syncWallet` method?